target/linux/ramips/mt7621/base-files/lib/preinit/04_handle_checksumming

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+do_checksumming_disable() {
+	. /lib/functions.sh
```

Comment:
This should be unnecessary as well, as it's sourced in the file where preinit files are sourced:
https://github.com/openwrt/openwrt/blob/master/package/base-files/files/etc/preinit